web-animations/timing-model/animations/updating-the-finished-state.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>Tests for updating the finished state of an animation</title>
+<link rel="help" href="https://w3c.github.io/web-animations/#updating-the-finished-state">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="../../testcommon.js"></script>
+<body>
+<script>
+'use strict';
+
+function validateFinishEvent(t, event, animation) {
+  t.step(function() {
+    assert_equals(event.target, animation, "Animation should be event target");
+    assert_times_equal(event.currentTime, animation.currentTime, "Event currentTime should be animation currentTime");
+    assert_times_equal(event.timelineTime, animation.timeline.currentTime, "Event timelineTime should be animation timeline's currentTime");
```

Comment:
Judging by http://w3c.github.io/web-animations/#animation-event-parameters, event timeline time should always be equal to the animation's start time plus the event current time. I've rewritten the assertions to conform to this.